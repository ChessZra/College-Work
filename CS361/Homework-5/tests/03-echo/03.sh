#!/bin/bash

PORT=$@

curl -s --user-agent "curl/XXX" http://127.0.0.1:$PORT/echo | sed s/$PORT/XXXXX/g
