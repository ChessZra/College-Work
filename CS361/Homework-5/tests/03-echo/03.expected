Host: 127.0.0.1:XXXXX
User-Agent: curl/XXX
Accept: */*